Readme update (#4964)
openhab/openhab2-addons: README.md

# openHAB 2 Add-ons

This repository contains the official set of add-ons that are implemented on top of openHAB 2 Core APIs.
Add-ons that got accepted in here will be maintained (e.g. adapted to new core APIs)
by the [openHAB 2 maintainers](https://github.com/orgs/openhab/teams/2-x-add-ons-maintainers).

> To get started with binding development, follow our guidelines and tutorials over at https://www.openhab.org/docs/developer/.

Reviewer: URL without link-syntax.

Author: Yes they are on purpose, as I want to raise awareness of how our link structure looks like. 

Especially for the other-addon-repositories, so that a reader can draw conclusions on where to look for more addon repos.

Reviewer: Thanks for clarification. :+1:

> If you are interested in openHAB core development, we invite you to come by on https://github.com/openhab/openhab-core.

Reviewer: URL without link-syntax.

> If you are interested in openHAB core development, we invite you to come by on https://github.com/openhab/openhab-core.

Reviewer 2: ```suggestion
If you are interested in openHAB 2 Core development, we invite you to come by on https://github.com/openhab/openhab-core.
```

## Add-ons in other repositories

> Some add-ons are not in this repository, but still part of the official [openHAB 2 distribution](https://github.com/openhab/openhab-distro). An incomplete list of other repositories follows below:

Reviewer 2: ```suggestion
Some add-ons are not in this repository, but still part of the official [openHAB 2 distribution](https://github.com/openhab/openhab-distro).
An incomplete list of other repositories follows below:
```

> * https://github.com/openhab/org.openhab.binding.zwave

Reviewer: URL without link-syntax.
Same for the two following lines.

* https://github.com/openhab/org.openhab.binding.zigbee
* https://github.com/openhab/openhab-webui

## Build

> openHAB 2 addons are [Java](https://en.wikipedia.org/wiki/Java_(programming_language)) `.jar` files.

Reviewer 2: ```suggestion
openHAB 2 add-ons are [Java](https://en.wikipedia.org/wiki/Java_(programming_language)) `.jar` files.
```

> The openHAB 2 buildsystem is based on [maven](https://maven.apache.org/what-is-maven.html).

Reviewer 2: ```suggestion
The openHAB 2 build system is based on [maven](https://maven.apache.org/what-is-maven.html).
```

The official IDE (Integrated development environment) is Eclipse.

To build all add-ons from the command-line, type in:
`mvn clean install`

> If you prefer an IDE, follow the instructions on https://www.openhab.org/docs/developer/development/ide.html

Reviewer: URL without link-syntax.
